extension/DataX/doriswriter/doc/doriswriter.md

```diff
@@ -163,6 +163,18 @@ DorisWriter 通过Doris原生支持Stream load方式导入数据， DorisWriter
   - 描述：每批次数据包含多行，每行为 Json 格式，每行的的分隔符即为 lineDelimiter。
   - 必选：否
   - 默认值：`\n`
+  
+* **format**
+
+  - 描述：导入数据的格式, 可以使是json或者csv。
+  - 必选：否
+  - 默认值：`json`
+  
+* **columnSeparator**
+
+  - 描述：当导入的格式是csv时, 字段之间的分隔符。
+  - 必选：否
+  - 默认值：`\t`
```

Comment:
There may be any character exist in user data, such as `\t` or `,`.
So it is better to use a invisible multi-bytes separator as default column separator here, I recommend to use `\x0707`

Reply:
already support invisible multi-bytes, like '\x01\x02'. but in version 0.14,  doris seem to only accept one byte. Branch master looks to handle multi-bytes .  ^_^

Reply:
You mean like this ?
```
list.add((value == null || value.equals("\\N")) ? "" : value.toString());
```

Reply:
OK I see. So please modify the description to remind this risk.